src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs

```diff
@@ -29,8 +30,15 @@ public class DisplayVersion
             string additionalInfo,
             bool isValidVersion = true,
             bool isCurrentInstalled = false,
-            bool autoReferenced = false)
+            bool autoReferenced = false,
+            string versionFormat = "N")
         {
+            if (versionFormat == null)
```

Comment:
oh, I see. no one should provide an explicit null here , but i guess its fine to check here.